pom.xml

```diff
@@ -98,6 +98,13 @@
         <version>${version.org.jgrapht}</version>
       </dependency>
 
+      <!-- servlet dependencies -->
+      <dependency>
+        <groupId>org.jboss.spec.javax.servlet</groupId>
+        <artifactId>jboss-servlet-api_3.0_spec</artifactId>
```

Comment:
I used jboss-servlet-api_3.0_spec as it is also available for EE6, but you are right, I could have used 3.1 which is used with EE7 and the tests are written for hal used with WildFly 10+ and EAP 7 which both implement EE7. As I am not using any Servlet 3.1 specific functionalities, I would say, it is OK to leave it as it is.